packages/tracker/tracker.d.ts

```diff
@@ -109,6 +109,16 @@ export namespace Tracker {
     }
   ): Computation;
 
+  /**
+   * Helper function to make the tracker work with promises.
+   * @param computation Computation that tracked
+   * @param func async function that needs to be called and be reactive
+   */
+  function withComputation<T>(
+    computation: Computation,
```

Comment:
It should say `Computation | null`, as otherwise it won't be isomorphic. Also, `currentComputation` should have the same type (it's `Computation` now).